primefaces/src/main/resources/META-INF/resources/primefaces/tabview/tabview.js

```diff
@@ -305,9 +305,58 @@ PrimeFaces.widget.TabView = PrimeFaces.widget.DeferredWidget.extend({
         })
         .on('keydown.tabview', function(e) {
             var element = $(this);
+            var keyCode = e.code;
             if(PrimeFaces.utils.isActionKey(e) && !element.hasClass('ui-state-disabled')) {
                 $this.select(element.index());
                 e.preventDefault();
+            } else {
+                switch(keyCode) {
+                    // Right arrow key
+                    case 'ArrowRight':
+                        var nextTab = element.nextAll('.ui-tabview-nav:not(.ui-state-disabled)').first();
+                        if(nextTab.length) {
+                            nextTab.trigger('focus.tabview');
+                        }
+                        e.preventDefault();
+                        break;
+                    // Left arrow key
+                    case 'ArrowLeft':
+                        var prevTab = element.prevAll('.ui-tabview-nav:not(.ui-state-disabled)').first();
+                        if(prevTab.length) {
+                            prevTab.trigger('focus.tabview');
+                        }
+                        e.preventDefault();
+                        break;
+                    // Space and Enter key
+                    case 'NumpadEnter':
```

Comment:
You need regular `Enter` as well as `NumpadEnter`  because they are treated separately.

Reply:
Done!